REVERSE_LC.php

```diff
@@ -402,7 +397,7 @@ public function authorizeUser() {
 				user can see my site data -- including all patient rows from all sites
 		*/
 		$this->getUser();
-		
+		$this->user->authorized = true; return; //KAREN
```

Comment:
Should we change this before pushing to production?

Reply:
> Should we change this before pushing to production?

I put my name to remember to ask about that, but I totally forgot. Thanks for catching that. I've added a new comment. I'm just returning the user as authorized because I'm still not clear on what should be the authentication data and the access levels.